network/onedrive/onedrive.SlackBuild

```diff
@@ -24,7 +24,7 @@
 cd $(dirname $0) ; CWD=$(pwd)
 
 PRGNAM=onedrive
-VERSION=${VERSION:-2.4.16}
+VERSION=${VERSION:-2.4.17}
```

Comment:
Could you fix the man pages so they are installed with ownership root:root and permissions 0644 please?

```
# stat -c '%a %n' /usr/man/man1/onedrive.1.gz
755 usr/man/man1/onedrive.1.gz
```